src/test/tests_common.h

```diff
@@ -437,7 +445,8 @@ tc_cli_start_basic(char *local_group_name, char *srv_group_name,
 void
 tc_srv_start_basic(char *srv_group_name, crt_context_t *crt_ctx,
 		   pthread_t *progress_thread, crt_group_t *grp,
-		   uint32_t *grp_size)
+		   uint32_t *grp_size, crt_init_options_t *init_opt,
```

Comment:
is there a need for 2 params here? Could we just say that if init_opt == NULL, that is same as 'use_init_opt = false'?